packages/scripts/src/monitor-dvm/common.ts

```diff
@@ -0,0 +1,104 @@
+import { getChainIdByUrl, getLatestBlockNumberByUrl } from "../utils/utils";
+import { VotingV2Ethers } from "@uma/contracts-node";
+import { delay } from "@uma/financial-templates-lib";
+import { BigNumber, utils } from "ethers";
+
+interface BotModes {
+  unstakesEnabled: boolean;
+  stakesEnabled: boolean;
+  governanceEnabled: boolean;
+  deletionEnabled: boolean;
+  emergencyEnabled: boolean;
+  rolledEnabled: boolean;
+  governorTransfersEnabled: boolean;
+  mintsEnabled: boolean;
+}
+
+export interface MonitoringParams {
+  jsonRpcUrl: string;
+  chainId: number;
+  startingBlock: number;
+  endingBlock: number;
+  pollingDelay: number;
+  botModes: BotModes;
+}
+
+export const initCommonEnvVars = async (env: NodeJS.ProcessEnv): Promise<MonitoringParams> => {
```

Comment:
i really like this pattern, but prefer you would have parsed and validated all values here rather than leaving out `UNSTAKE_THRESHOLD` and others to be referenced in the functions rather than the monitoring params

Reply:
fixed in 228982dc457e606217a9faca1abc24c0e5d5ff14